src/webview.ts

```diff
@@ -3,21 +3,36 @@ import * as path from "path";
 import * as fs from "fs";
 import OpenAI from "openai";
 
+const apiKey = vscode.workspace.getConfiguration().get<string>("openAI.apiKey");
+
+if (!apiKey) {
+  vscode.window.showErrorMessage("Open API Key is missing.");
+}
+
 // VSCode의 settings.json에서 API 키를 가져옵니다.
 const openai = new OpenAI({
-  apiKey: vscode.workspace.getConfiguration("codingRoadmapAssistant").get<string>("openaiApiKey"),
+  apiKey,
 });
 
 // GPT API 호출 함수
 async function main(prompt: string) {
```

Comment:
Should rename this function as it is not the "main" function.

Suggestion: `getGPTResponse()`

Reply:
agree